agent/worker_test.go

```diff
@@ -437,6 +531,14 @@ func TestWorkerUpdate(t *testing.T) {
 					},
 					Action: api.AssignmentChange_AssignmentActionRemove,
 				},
+				{
```

Comment:
Since this test is testing removing non-existing items as well, do we also want to attempt to remove `config-2` to ensure that it doesn't fail?